configure.ac

```diff
@@ -1464,21 +1464,19 @@ else
 fi
 
 # ---------------------------------------------------------------------------
-# Configuring pmi, pmilib, and pm
+# Configuring pmi and pm
 #
 
 AC_ARG_WITH(pmi, [
   --with-pmi=name - Specify the pmi interface for MPICH.
-                      pmi1 - use PMIv1 (default)
-                      pmi2 - use PMI2
-                      pmix - use PMIx
-],, with_pmi=pmi1)
+                      mpich     - use MPICH's PMI library (default)
+                      install   - use MPICH's PMI library in embedded mode
```

Comment:
`install` means we do not embed, actually.

Reply:
Ah yes. This was a little messed up, sorry. Revised this to offer the options `embedded` (instead of `mpich`) and `install` - as proposed by Hui above.